landing/index.html

```
@@ -868,24 +881,30 @@ <h2 class="section-heading">Three steps</h2>
           <div class="setup-num">01</div>
           <p class="setup-text">Install from PyPI</p>
           <div class="setup-code" onclick="copyText(this, 'pip install brainlayer')">
-            <span>pip install brainlayer</span>
-            <svg class="copy-icon" width="14" height="14" viewBox="0 0 16 16" fill="none" stroke="currentColor" stroke-width="1.5"><rect x="5" y="5" width="9" height="9" rx="1.5"/><path d="M5 11H3.5A1.5 1.5 0 012 9.5v-6A1.5 1.5 0 013.5 2h6A1.5 1.5 0 0111 3.5V5"/></svg>
+            <code>pip install brainlayer</code>
+            <button class="copy-btn" aria-label="Copy">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)